mwdb/model/migrations/versions/769c06a5f62c_openid_tables.py

```diff
@@ -0,0 +1,62 @@
+"""OpenID tables
+
+Revision ID: 769c06a5f62c
+Revises: 3648d869ef45
+Create Date: 2021-09-06 12:37:40.325947
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "769c06a5f62c"
+down_revision = "35916645cf47"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "openid_provider",
+        sa.Column("id", sa.Integer(), autoincrement=True, nullable=False),
+        sa.Column("name", sa.String(length=64), nullable=False),
+        sa.Column("client_id", sa.String(length=64), nullable=False),
+        sa.Column("client_secret", sa.String(length=64), nullable=False),
```

Comment:
Actually... `client_secret` can be null if we use public key cryptography for signing JWT (RS256 + JWKS).
This method will be probably used with Keycloak and AFAIK is more common in real-life providers than shared secret.

https://auth0.com/docs/security/tokens/json-web-tokens/json-web-key-sets